Content.Server/Tesla/EntitySystem/LightningArcShooterSystem.cs

```diff
@@ -32,12 +32,16 @@ public override void Update(float frameTime)
         var query = EntityQueryEnumerator<LightningArcShooterComponent>();
         while (query.MoveNext(out var uid, out var arcShooter))
         {
+            // SS220-SM-begin
+            if (!arcShooter.Enabled)
+                continue;
+            // SS220-SM-end
             if (arcShooter.NextShootTime > _gameTiming.CurTime)
                 continue;
 
             ArcShoot(uid, arcShooter);
             var delay = TimeSpan.FromSeconds(_random.NextFloat(arcShooter.ShootMinInterval, arcShooter.ShootMaxInterval));
-            arcShooter.NextShootTime += delay;
+            arcShooter.NextShootTime = _gameTiming.CurTime + delay; // SS220-SM-Arc-Fix
```

Comment:
Хочу пометочку такого вида
```suggestion
            // SS220-SM-begin
            //arcShooter.NextShootTime += delay;
            arcShooter.NextShootTime = _gameTiming.CurTime + delay;
            // SS220-SM-end
```